src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -316,6 +316,7 @@ public void IOExceptionRaisedCreatingSarifLog()
                         TargetFileSpecifiers = new string[] { GetThisTestAssemblyFilePath() },
                         OutputFilePath = path,
                         Verbose = true,
+                        Force = true
```

Comment:
>Force [](start = 24, length = 5)

Now that `AnalyzeCommandBase` respects the `force` option, several tests that relied on their ability to blithely overwrite the output file started failing.